syft/workers/base.py

```diff
@@ -1273,3 +1273,12 @@ def force_detail(worker: AbstractWorker, worker_tuple: tuple) -> tuple:
                 worker.object_store.rm_obj(obj.id)
 
         return result
+
+    @classmethod
```

Comment:
Would it make sense to offer this as an instance method? Seems like it could be nice to be able to write something like `alice.is_framework_supported("tensorflow")`.

Reply:
classmethod  will allow the method to be invoked via class and its instance. So, both these statements will be valid : 

- ```
- alice = VirtualWorker(id="alice", hook=hook, verbose=False)
- alice.is_framework_supported("tensorflow")
- ```
- 
- `VirtualWorker.is_framework_supported("tensorflow")`